packages/design-system/src/components/timeline/index.tsx

```diff
@@ -103,6 +103,26 @@ const Timeline = ({
     }
   }, [containerRef, bidders, auctionTimeout]);
 
+  useEffect(() => {
+    const _zoom = zoomLevel < 1 ? 1 : zoomLevel;
+    setZoom(_zoom);
+    setTimeoutBlockWidth(scrollWidth - auctionTimeout * _zoom);
+  }, [scrollWidth, auctionTimeout, zoomLevel]);
+
+  useEffect(() => {
+    const timeout = setTimeout(() => {
+      setAnimate(true);
+    }, 100);
+
+    return () => clearTimeout(timeout);
+  }, []);
+
+  useEffect(() => {
+    if (containerRef.current) {
+      setScrollWidth(containerRef.current.scrollWidth);
+    }
+  }, [containerRef, bidders, auctionTimeout]);
+
```

Comment:
Why are we doing it again?

Reply:
Probably a bad merge conflict got through. Removed in recent commits.